src/Lesson4/HomeTask10.java

```diff
@@ -0,0 +1,22 @@
+package Lesson4;
+
+public class HomeTask10 {
+int num1;
+String text;
+int num2;
+        // ...
+       public void OverloadedMethod(String num1)
+        {
+            // ...
+        }
+
+    public void OverloadedMethod(int num1, String num2)
+        {
+
+        }
+
+    public void OverloadedMethod(int num1,int text, int num2)
```

Comment:
"Write another class to demonstrate the concept of methods overloading."
I don't see another class with demonstration.